warehouse/models/staging/stg_waste_emission.sql

```diff
@@ -0,0 +1,227 @@
+{% set cnd_split = 0.5 %}
```

Comment:
will be usedlater in code for  splitting the waste tonnage between different categories.